plugin/plugins/bilibili_danmaku/__init__.py

```
@@ -667,6 +667,12 @@ async def push_danmaku_tick(self, **_):
     # AI 可调用入口
     # ==========================================
 
+    def _get_connection_info(self) -> dict:
+        """获取连接详情（内部方法）"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)